### DISCUSSION TITLE
The lastest release is not on Zenodo

### DISCUSSION THREAD

**Discussion by Ovewh** [metno/pyaerocom#697] [General]
Hi,
I noticed the version of pyaerocom on [Zenodo](https://zenodo.org/record/4558810) is quite old. Isn't it supposed to be kept up to date?

**Comment by lewisblake** [comment]
Moreover, shouldn't the Zenodo badge be in the README ;)

**Comment by jgriesfeller** [comment]
Met does not allow us to integrate Zenodo in our github workflow (Zenodo integration needs access to the gihub repo, but our IT department does not want to allow that due to security concerns). So getting a DOI is handwork unfortunately

**Comment by avaldebe** [comment]
Is this an issue? Or a discussion?

**Comment by lewisblake** [comment]
Probably the latter, so I converted it. Anyway, I looks like Jonas is the person with this ability, unless @jgriesfeller or someone else can do it. I agree the DOI should be kept up-to-date though
